tests/test_generators/python/test_python_ifabsent_processor.py

```diff
@@ -374,6 +374,44 @@ def test_uri_default_value():
     )
 
 
+# If this function does not get fixture scoped to the module the
+# PythonIfAbsentProcessor instance suddenly has some attributes
+# coming from tests using the kitchen-sink example! Invalidating that
+# way the results of the test.
+@pytest.fixture(scope="module")
```

Comment:
```suggestion
```

Seems to work to me except for the assert on the forward references dict! and shouldn't be a fixture anyway :)

Reply:
Removing it, since the issue somehow magically has disappeared... And the fix was non-sense nevertheless.